pipeline_steps/common.groovy

```diff
@@ -188,7 +188,11 @@ def conditionalStage(Map args){
   stage(args.stage_name){
     if (env.STAGES.contains(args.stage_name)){
         print "Stage Start: ${args.stage_name}"
-        args.stage()
+        ansiColor('xterm') {
```

Comment:
While those are indeed duplicate calls, I think they are worth leaving in as those functions may be used outside of a conditional step or stage.